### DISCUSSION TITLE
I cannot request my application

### DISCUSSION THREAD

**Discussion by Rashik36** [community/community#84190] [GitHub Education]
### Hi everyone,

I applied for the student benefits using my college email, github automatically knew my college and asked for my id card picture, I provided the picture and hit submit, then it told me I needed to enable 2-factor authentication and update my billing information. I did both and tried again, but this time it got stuck in the processing information. it says it can take up several seconds so I just let it be for about an hour and nothing changed. I tried it several times more, restarted my browser, used incognito mode, but nothing.
<img width="525" alt="Screenshot 2024-01-05 150719" src="https://github.com/community/community/assets/118498377/96a96364-fb3a-4535-a389-bd10a49bc616">

**Comment by rmitaccs40** [comment]
I have same problems, I wonder what can I do?

**Comment by Rashik36** [comment]
I have spend all day trying different things, nothing works, it worked the
first time, then for some reason it just doesn’t now

**Comment by InvisiblePro** [comment]
Heyy all there!
It requires having a date i.e. 10-Oct-20xx mentioned on the Identity proof showing that you are in the mentioned institution. It will take time of at least 1 week time after you submit your documents/proofs. Make sure that the Date includes -- month and year.
Have a good journey ✨

Check it out!
Provide your review after this step
If this worked, please make sure to "mark the answer".

**Reply by d1mo22** [reply]
My document has the date and it won't let me either, it keeps loading for hours.

**Comment by d1mo22** [comment]
Is there a solution or is it a problem with the site?

**Comment by Rashik36** [comment]
Exactly, mine also has the date, it is sideways so that might be an issue,
but it keeps loading, if the request went in and got rejected then I would
understand. But it’s not even letting me issue an request

**Reply by d1mo22** [reply]
If it has been accepted supposedly a screen should appear saying that it has been sent. But I don't see anything either 🥲.

**Comment by Rashik36** [comment]
Same it just saying processing information forever

**Comment by d1mo22** [comment]
I have managed to send, you have to take the picture with the device and then you will get a blue message saying that you can send to the request. I don't know why but it doesn't work to upload the photos from the computer/mobile.

**Reply by rmitaccs40** [reply]
oh directly from github desktop?

**Reply by rmitaccs40** [reply]
Finally works thank u.
